android/app/build.gradle

```diff
@@ -34,18 +40,39 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "com.example.dart_wormhole_gui"
+        applicationId "com.leastauthority.dart_wormhole_gui"
         minSdkVersion 16
         targetSdkVersion 30
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
     }
 
+    signingConfigs {
+       release {
+           keyAlias keystoreProperties['keyAlias']
+           keyPassword keystoreProperties['keyPassword']
```

Comment:
It's weird that the key password had to be retrieved from the Properties object.. if there is a way to let the build prompt for this, it would be much better.

Reply:
Good point. After quick search, one option is to take form ENV variables
https://stackoverflow.com/questions/20562189/sign-apk-without-putting-keystore-info-in-build-gradle
Or do some coding, to build such input?
https://gyulajuhasz.com/blog/dynamic-password-input-with-gradle/

Reply:
@ewanas I made change at least to move out key.properties file from project dir. Usage as ENV parameters is not good alternatives, also it may not work with Android Studio (according others)